docs/common/migrations_server.md

```diff
@@ -8,7 +8,7 @@ services to read from:
 sqldelight {
   databases {
     Database {
-      migrationOutputDirectory = file("$buildDir/resources/main/migrations")
+      migrationOutputDirectory = layout.buildDirectory.dir("resources/main/migrations")
```

Comment:
I'm not sure if I should also update this in the gradle plugin? 

https://github.com/cashapp/sqldelight/blob/7a1f7597c6f90f492c33632a43956c335bbd5391/sqldelight-gradle-plugin/src/test/schema-output/build.gradle#L8-L13?

Reply:
Feel free to do so in a follow-up PR!